Changes between KDE 3.5.10 tag and KDE 3.5 branch r892605.
Only looking at subdirectory kdebase/nsplugins here.
Logs since r849627 of the branch but this might be wrong.

r892605 | terwou
backport r892601 Fix load plugins on demand
  M nspluginloader.cpp
  M nspluginloader.h


Index: nspluginloader.cpp
===================================================================
--- kdebase/nsplugins/nspluginloader.cpp	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdebase/nsplugins/nspluginloader.cpp	 (.../branches/KDE/3.5)	(revision 892605)
@@ -69,7 +69,7 @@
     if (cfg.readBoolEntry("demandLoad", false)) {
         _button = new QPushButton(i18n("Start Plugin"), dynamic_cast<EMBEDCLASS*>(this));
         _layout->addWidget(_button, 0, 0);
-        connect(_button, SIGNAL(clicked()), this, SLOT(doLoadPlugin()));
+        connect(_button, SIGNAL(clicked()), this, SLOT(loadPlugin()));
         show();
     } else {
         _button = 0L;
@@ -84,11 +84,15 @@
     }
 }
 
+void NSPluginInstance::loadPlugin()
+{
+    delete _button;
+    _button = 0;
+    doLoadPlugin();
+}
 
 void NSPluginInstance::doLoadPlugin() {
-    if (!inited) {
-        delete _button;
-        _button = 0L;
+    if (!inited && !_button) {
         _loader = NSPluginLoader::instance();
         setBackgroundMode(QWidget::NoBackground);
         WId winid = stub->winId();
Index: nspluginloader.h
===================================================================
--- kdebase/nsplugins/nspluginloader.h	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdebase/nsplugins/nspluginloader.h	 (.../branches/KDE/3.5)	(revision 892605)
@@ -55,6 +55,7 @@
     void javascriptResult( int id, QString result ) { stub->javascriptResult( id, result ); }
 
 private slots:
+    void loadPlugin();
     void doLoadPlugin();
 
 protected:
